indico/modules/events/registration/client/js/form/fields/DateInput.jsx

```diff
@@ -29,6 +29,8 @@ export function DateInputComponent({
   required,
   dateFormat,
   timeFormat,
+  min,
+  max,
```

Comment:
Does some customization in the UN plugin add an option to set a min/max date? Because the core does not have this, so this would not be needed in here.

Reply:
I have no strong opinions regarding this, but I think it could be useful for plugins to be able to reuse the component and add those fields instead of wrapping it in a new component. Again, no strong opinions. Let me know before I make the rest of the changes, though. :)

Reply:
Vu has just confirmed this way works better on our side.